contracts/src/Shadowlings.sol

```diff
@@ -10,86 +10,86 @@ import {Verifier as RecoveryVerifier} from "./verifiers/recovery/Verifier.sol";
 import {Verifier as RegisterVerifier} from "./verifiers/register/Verifier.sol";
 
 contract Shadowlings is IAccount, Verifier {
+    bytes32 private constant _R = bytes32(uint256(keccak256("Shadowlings.r")) - 1);
```

Comment:
First valid `r` value close to `keccak256("Shadowlings.r")`. Using `keccak256("Shadowlings.r")` directly is not a valid X-coordinate on `secp256k1`.